internal/utils/datasource.go

```diff
@@ -61,7 +61,7 @@ resources:
 {{- end }}
 `
 
-const kustomizeWithPatchesTemplateString = `
+const kustomizeWithPatchesTemplateString = `---
 apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Now we'll have an empty line at the begging of the file, we need to remove it.

Reply:
please can yo elaborate? "proper" yaml suggests to start with `---` as first line.

Reply:
Sorry want i wanted to suggest is to just have `apiVersion` as the first line instead of `---`. There is no `---` when you run `kustomize create`. We should be using the format Kustomize uses.